src/Core/Data/BrewClubRoleManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public static class BrewClubRoleManager
+	{
+
+		public static string Admin = "Admin"; // has rights to all functions
+		public static string Moderator = "Moderator"; // has rights to moderation functions
+		public static string Member = "Member"; // allowed to post content
+		public static string User = "User"; // allowed to comment on comment and maintain a profile
+
+	}
```

Comment:
This file was originally named RoleNames, and I renamed it because I'm going to put the role name constants into the class we'll use for managing roles.  I haven't created said class yet though.